### ISSUE TITLE
Arduino Library Specification

### ISSUE THREAD

**Issue by GoogleCodeExporter** [olikraus/ucglib#35]
```

Create Arduino compatible u8glib library
https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification


Publish ucglib:
https://github.com/arduino/Arduino/wiki/Library-Manager-FAQ
https://github.com/arduino/Arduino/issues


```

Original issue reported on code.google.com by `olikr...@gmail.com` on 11 Sep 2014 at 5:00

**Comment by olikraus** [comment]
Almost done. Only tests required for the new zip file.

**Comment by olikraus** [comment]
problem is the visibility of the examples. they are only available after restart under the library section.

**Comment by olikraus** [comment]
visibility fixed in 1.6.5
